desc/compute/_core.py

```diff
@@ -3037,151 +2837,6 @@ def _theta_PEST_t(params, transforms, profiles, data, **kwargs):
     return data
 
 
-@register_compute_fun(
-    name="theta_PEST_tt",
```

Comment:
I would prefer to keep stuff like this as independent quantities. For stuff like `g_rt` and `g_tr`, those are the same both physically and numerically, independent of how we parameterize or discretize things.

Stuff like `theta_PEST_tt` and `lambda_tt` are actually different things physically, its just that they happen to the same numerically because we define `theta_PEST = theta + lambda`, but in theory other relations are possible and equally valid physically.